libraries/botbuilder-ai/src/index.ts

```diff
@@ -11,7 +11,8 @@ export * from './geographyV2';
 export * from './instanceData';
 export * from './intentData';
 export * from './luisRecognizer';
+export * from './luisTelemetryConstants';
```

Comment:
This is parity with .NET: https://github.com/microsoft/botbuilder-dotnet/blob/20b1539f4eea4dea0bf184ca6e75b6ae6f033748/libraries/Microsoft.Bot.Builder.AI.LUIS/LuisTelemetryConstants.cs#L9